microsoft/PowerToys
# [Telemetry] Add basic interaction events to FZ

## Summary of the Pull Request

**What is this about:**
Adds some basic interaction events to FancyZones, such as:
- Starting the drag Window movement. `FancyZones_MoveSizeEnter`
- A newly created window snapping to a zone. `FancyZones_MoveNewWindowIntoZone`
- Using the Keyboard to snap or extend to a zone. `FancyZones_KeyboardSnapToZone`

**What is include in the PR:** 
The described telemetry events.

**How does someone test / validate:** 
@SeraphimaZ  check to see if the places make sense for the events.
@mykhailopylyp for second pair of eyes on telemetry code review.

## Quality Checklist

- [ ] **Linked issue:**

## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
